react/features/local-recording/reducer.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+
+import { ReducerRegistry } from '../base/redux';
+import {
+    LOCAL_RECORDING_ENGAGED,
+    LOCAL_RECORDING_STATS_UPDATE,
+    LOCAL_RECORDING_TOGGLE_DIALOG,
+    LOCAL_RECORDING_UNENGAGED
+} from './actionTypes';
+import { recordingController } from './controller';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+ReducerRegistry.register('features/local-recording', (state = {}, action) => {
+    logger.debug(`Redux state (features/local-recording):\n ${
```

Comment:
I recommend removing this logger statement as the reducer can fire many times a second. I assume it'll be really noisy on mobile which during development may log messages onto the screen.